libs/template/helpers.go

```diff
@@ -46,6 +47,10 @@ func loadHelpers(ctx context.Context) template.FuncMap {
 		"regexp": func(expr string) (*regexp.Regexp, error) {
 			return regexp.Compile(expr)
 		},
+		// Alias for https://pkg.go.dev/math/rand#Intn. Returns, as an int, a non-negative pseudo-random number in the half-open interval [0,n).
+		"randIntn": func(n int) int {
```

Comment:
Other functions are snake cased, this should be too (as well as self-explanatory). For ex: `random_int`.

Reply:
Renamed!